src/auto_scheduler/compute_dag.cc

```diff
@@ -969,6 +969,9 @@ void ComputeDAG::RewriteLayout(const Array<Step>& transform_steps) {
       }
     }  // end for placeholder
   }    // end for stage
+  p_dag->access_analyzer = AccessAnalyzer(p_dag->tensors);
+  p_dag->ops = p_dag->access_analyzer->ops_topo_order;
+  p_dag->flop_ct = FlopEstimator().EstimateFlop(p_dag->ops);
```

Comment:
Why does it work before without the fix?

Reply:
I would like to know the answer too... For the ApplySteps in compute DAG, previously we simply put all non-placeholder ops to the list and take the last op (assuming it is the only output) to create a new schedule. After using `access_analyzer.is_output` to check what ops should be put to the list, `is_output` throws exceptions due to the out-of-date `is_output` map. That's why I found this issue. Maybe most use cases so far don't enable layout rewrite?